src/resources/about-food-labels/view.html

```diff
@@ -84,7 +82,7 @@ <h1>
         <p>
           If the name of the food does not reflect its true nature, then you
           must include a description of the true nature on the label. For
-          example, Luke's Hot's Sauce (red chilli pepper sauce).
+          example, Luke’s Hot’s Sauce (red chilli pepper sauce).
```

Comment:
should we be using the html &rsquo; ?

Reply:
```&rsquo;``` *

Reply:
@duttonw  According to: https://www.w3.org/International/questions/qa-escapes

"It is almost always preferable to use an encoding that allows you to represent characters in their normal form, rather than using named character references or numeric character references."

In the past I have escaped these characters but documents like this seem to suggest you only escape when a browser won't understand it.